doc/source/reference/commands/object_inspect.rst

```diff
@@ -747,6 +737,42 @@ The max value of each space is the max number of chunks. See
 :ref:`object-inspect-return-value-index-column-value-statistics-total-chunk-size`
 for the max number of chunks.
 
+.. _object-inspect-return-value-index-column-value-statistics-next-physical-segment-id:
+
+``INDEX_COLUMN_VALUE_STATISTICS_NEXT_PHYSICAL_SEGMENT_ID``
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. versionadded:: 9.0.2
+
+This value the ID of the segment to the inspected index column use next.
+
```

Comment:
It can be simplified?

`This value is the ID of the segment. The inspected index column use it as the next segment ID.`